### ISSUE TITLE
provide a way to load the openapi.yaml that doesn't server it at an endpoint

### ISSUE THREAD

**Issue by mmerickel** [Pylons/pyramid_openapi3#174]
Due to issues like https://github.com/Pylons/pyramid_openapi3/issues/104 I cannot define a yaml that fully matches the deployment environment so I'd prefer to just not serve the yaml if I can help it. I'd just like it to be used to validate paths, as the yaml defines everything relative to the app's subpath so works across deployments so long as pyramid_openapi3 is using `request.application_url` as the base path.

**Comment by zupo** [comment]
Makes sense and also seems like it won't complicate the codebase much, so I'd probably be +1 on a PR to introduce this.
